modules/rollback-operator-upgrades-console.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * upgrade/upgrade-operator.adoc
+:_content-type: PROCEDURE
+[id="rollback-operator-upgrades-console_{context}"]
+= Rolling back an Operator upgrade by using the web console
+
+You can roll back the Operator version by using the {ocp} web console.
+
+.Prerequisites
+
+- You have access to an {ocp} cluster web console using an account with `cluster-admin` permissions.
+
+.Procedure
+
+. Navigate to the *Operators* -> *Installed Operators* page.
+. Locate the {product-title-short} Operator and click on it.
+. On the *Operator Details* page, select *Uninstall Operator* from the *Actions* list. Following this action, the Operator stops running and no longer receives updates.
+. Determine the previous version you want to roll back to by choosing one of the following options:
+* If the current Central instance is running, you can query the {product-title-short} API to get the rollback version by running the following command from a terminal window:
++
+[source,terminal]
+----
+$ curl -k -s -u <user>:<password> https://<central hostname>/v1/centralhealth/upgradestatus | jq -r .upgradeStatus.forceRollbackTo
+----
+* You can create a pod to extract the previous version by performing the following steps:
++
+[NOTE]
+====
+This procedure can only be used for {product-title-short} release 3.74 and earlier when the `rocksdb` database is installed.
+====
+.. Navigate to *Workloads* -> *Deployments* -> *central*.
+.. Under *Deployment details*, click the down arrow next to the pod count to scale down the pod.
+.. Navigate to *Workloads* -> *Pods* -> *Create Pod* and paste the contents of the pod spec as shown in the following example into the editor:
++
+
+[source,yaml,subs="attributes+"]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: get-previous-db-version
+spec:
+  containers:
+  - name: get-previous-db-version
+    image: registry.redhat.io/advanced-cluster-security/rhacs-main-rhel8:<rollback version>
+    command:
+    - sh
+    args:
+    - '-c'
+    - "cat /var/lib/stackrox/.previous/migration_version.yaml | grep '^image:' | cut -f 2 -d : | tr -d ' '"
+    volumeMounts:
+    - name: stackrox-db
+      mountPath: /var/lib/stackrox
+  volumes:
+  - name: stackrox-db
+    persistentVolumeClaim:
+      claimName: stackrox-db
+----
+. Click *Create*.
+. After the pod is created, click the *Logs* tab to get the version string.
```

Comment:
I believe these two items must be at the second level of indentation, not the first one since they belong to determining the previous version of rollback:

```suggestion
.. Click *Create*.
.. After the pod is created, click the *Logs* tab to get the version string.
```